packages/hyperion-autologging/src/ALSurfaceMutationPublisher.ts

```diff
@@ -58,7 +59,7 @@ export type InitOptions = Types.Options<{
 }>;
 
 export function publish(options: InitOptions): void {
-  const { domSurfaceAttributeName = AUTO_LOGGING_SURFACE, channel, flowletManager, cacheElementReactInfo, componentNameValidator = null } = options;
+  const { domSurfaceAttributeName = AUTO_LOGGING_SURFACE, channel, flowletManager, cacheElementReactInfo, componentNameValidator = defaultComponentNameValidator } = options;
```

Comment:
we assume `publish` is called once per app, so fair to say `componentNameValidator` is static. You can have a static `setComponentNameValidator` in info class and avoid reduce the size of that object.